src/test/librbd/test_librbd.cc

```diff
@@ -3099,6 +3099,192 @@ TEST_F(TestLibRBD, TestAioCompareAndWriteStripeUnitSuccess)
   rados_ioctx_destroy(ioctx);
 }
 
+TEST_F(TestLibRBD, TestAioCompareAndWritevIovecLenDiffers)
```

Comment:
Capital `V` since it's a shorthand for vectored or something like that:
```suggestion
TEST_F(TestLibRBD, TestAioCompareAndWriteVIovecLenDiffers)
```